src/Components/Shops/CreateShop.tsx

```diff
@@ -0,0 +1,46 @@
+import { Button, Form, Input, Typography } from 'antd';
+import { CiShop } from 'react-icons/ci';
+
+const CreateShop = () => {
+  return (
+    <div className="flex justify-center items-center w-screen h-screen">
+      <Form
+        name="create-shop"
+        style={{
+          backgroundColor: 'rgba(255, 255, 255, 0.33)',
+          boxShadow: '0px 8px 32px 0 rgba(0, 0, 255, 0.33)',
+          padding: '60px 60px 60px 60px',
+          borderRadius: '10px',
+        }}
```

Comment:
If possible, handle it with tailwind css.
may be something like below: 
`class="bg-white/30 shadow-[0_8px_32px_rgba(0,0,255,0.33)] p-16 rounded-lg"`